Content.Server/StationEvents/Components/GameDirectorStoriesComponent.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Content.Shared.chaos;
+
+namespace Content.Server.StationEvents.Components;
+
+[RegisterComponent, Access(typeof(GameDirectorSystem))]
+public sealed class GameDirectorStoriesComponent : Component
```

Comment:
rename to GameStories (don't forget filenames) (tho i *think* RoundStory makes more sense but i havent done a deep dive so)

Reply:
Stories aren't long enough to last an entire round. More like 30-60 minutes.